demos/ansi.c

```diff
@@ -125,12 +125,10 @@ ansi_to_markup( char *sequence, size_t length, markup_t *markup )
             else if( (set_fg == 0) && (code == 5) )
             {
                 set_fg = 1;
-                code = 0;
```

Comment:
This line prevents from ever entering the test just after. Do you know how the analyzer decides this can be safely removed ?